src/plugins/expression_error/public/plugin.ts

```diff
@@ -26,8 +26,10 @@ export class ExpressionErrorPlugin
   implements Plugin<ExpressionErrorPluginSetup, ExpressionErrorPluginStart, SetupDeps, StartDeps>
 {
   public setup(core: CoreSetup, { expressions }: SetupDeps): ExpressionErrorPluginSetup {
-    expressions.registerRenderer(errorRendererFactory(core));
-    expressions.registerRenderer(debugRendererFactory(core));
+    core.getStartServices().then(([start]) => {
+      expressions.registerRenderer(errorRendererFactory(start));
+      expressions.registerRenderer(debugRendererFactory(start));
+    });
```

Comment:
@elastic/kibana-presentation I am interested to get close attention to this. It is a pattern I repeat a few times in this PR. Is it OK to register the renderer async? An alternative would be to access the start services from the `render` function, which is already an async function. But doing it this way felt like less impact for some reason.

Reply:
I don't think this is causing any problems with the PR, and I am able to render Canvas elements using these expression renderers in this PR with no problems.

That said, I'd usually recommend _registering_ items synchronously, and accessing start services from the `render` method. You can see an example of how this was done in `src/plugins/chart_expressions/expression_gauge/public/plugin.ts`